OxfordBodleian/Juel-Jensen/BDLjj29.xml

```diff
@@ -10,15 +10,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="en">Fragment of the Miracles of Mary</title>
+                <!-- 
+                <title xml:lang="gez" xml:id="title1">ተአምረ፡ ማርያም፡</title>
+                <title xml:lang="gez" corresp="#title1" type="normalized">Taʾammǝra Māryām</title>
+                -->
+                <title xml:lang="en" xml:id="title1">Fragment of the Miracles of Mary</title>
```

Comment:
Maybe this could be updated to the precise miracle? For me, it would be fine to take the PEMM title as reference. So maybe  "Miracle of Mary: A young man who used to crown the icon of the Virgin Mary with roses (fragment)"?
https://pemm.princeton.edu/stories/153